rotkehlchen/db/schema.py

```diff
@@ -680,6 +680,10 @@
     manual_price INTEGER NOT NULL CHECK (manual_price IN (0, 1)),
     owner_address TEXT,
     blockchain TEXT GENERATED ALWAYS AS ("ETH") VIRTUAL,
+    is_lp INTEGER NOT NULL CHECK (is_lp IN (0, 1)),
```

Comment:
this is only for uniswap v3?

Reply:
Hmmm right now yeah, I guess so. Is it bad?

Reply:
no no just asking to understand the approach